lib/pages/chat/chat_pinned_events/pinned_messages.dart

```diff
@@ -200,10 +202,29 @@ class PinnedMessagesController extends State<PinnedMessages>
     }
   }
 
+  void _initPinnedEvents() {
+    if (widget.pinnedEvents.isNotEmpty) {
+      eventsNotifier.value = widget.pinnedEvents;
+    } else {
+      final currentRoomId = GoRouterState.of(context).pathParameters['roomid'];
```

Comment:
```suggestion
      final currentRoomId = GoRouterState.of(context).pathParameters['roomid'] ?? widget.timeline?.room;
```

Reply:
No need, because in the case we reload page `/rooms/!gNBHFsDrkWKBTZBkGT:linagora.com/pinnedmessages` then `widget.timeline?.room` is null.